test/e2e/utils/test_context.go

```diff
@@ -88,7 +88,7 @@ func (kc *KBTestContext) InstallCertManager() error {
 	if _, err := kc.Kubectl.Command("create", "namespace", "cert-manager"); err != nil {
 		return err
 	}
-	if _, err := kc.Kubectl.Command("label", "namespace", "cert-manager", "certmanager.k8s.io/disable-validation=true"); err != nil {
+	if _, err := kc.Kubectl.Command("label", "namespace", "cert-manager", "cert-manager.io/disable-validation=true"); err != nil {
```

Comment:
It seems we no longer need this step if using v0.11.* according to the cert-manager documentation (https://docs.cert-manager.io/en/latest/getting-started/install/kubernetes.html#installing-with-regular-manifests).

Reply:
Agreed, I'll update the version of K8s (used by kind) for the integration tests >1.15 so the validation step isn't needed as per cert-manager docs. 

> If you are running Kubernetes v1.15 or below, you will need to add the --validate=false flag to your kubectl apply command above else you will receive a validation error relating to the x-kubernetes-preserve-unknown-fields field in our CustomResourceDefinition resources. This is a benign error and occurs due to the way kubectl performs resource validation.

Reply:
I see this being needed based on the K8s version not the CertMan version so when running `test_e2e_v2` against  `<1.16` the test failed without this being set.  

Will re-run locally and double check.

Reply:
Totally right, local testing shows its not need anymore.